lib/galaxy/datatypes/blast.py

```diff
@@ -0,0 +1,281 @@
+"""NCBI BLAST datatypes.
+
+Covers the ``blastxml`` format and the BLAST databases.
+"""
+
+import logging
+import os
+from time import sleep
+
+from galaxy.datatypes.data import get_file_peek
+from galaxy.datatypes.data import Data, Text
+from galaxy.datatypes.xml import GenericXml
+
+log = logging.getLogger(__name__)
+
+
+class BlastXml(GenericXml):
+    """NCBI Blast XML Output data"""
```

Comment:
Separately I want to add a new ``blastxml2`` datatype to Galaxy, and have filed https://github.com/edamontology/edamontology/issues/200 about adding an EDAM tag for the NCBI BLAST XML2 format.